pages/content/amp-dev/documentation/components/index.html

```diff
@@ -22,12 +22,12 @@
 
 
 highlighted_components:
-  - /content/amp-dev/documentation/components/reference/amp-analytics.md
-  - /content/amp-dev/documentation/components/reference/amp-autocomplete.md
-  - /content/amp-dev/documentation/components/reference/amp-carousel.md
-  - /content/amp-dev/documentation/components/reference/amp-ad.md
-  - /content/amp-dev/documentation/components/reference/amp-list.md
-  - /content/amp-dev/documentation/components/reference/amp-story.md
+  - /content/amp-dev/documentation/components/reference/amp-analytics-v0.1.md
```

Comment:
I actually went back and forth on this, and found it a little nicer to have explicit versions in the directory. The serving path however omits the version for the latest version. If you feel strongly about this, feel free to chime in again :) But ideally it would be a subsequent PR.

Reply:
I concern was that if a version gets upgraded, we also need to change all internal links. Subsequent PR sounds fine.

Reply:
sg, thanks! On your primary concern, when a version gets upgraded, that should not happen in the future, as it should introduce a new doc (and the old doc should be kept around).